components/wmtk_components/wildmeshing/wmtk/components/wildmeshing/wildmeshing.cpp

```diff
@@ -171,10 +152,136 @@ void wildmeshing(const base::Paths& paths, const nlohmann::json& j, io::Cache& c
     const double bbdiag = (bmax - bmin).norm();
     const double target_edge_length = options.target_edge_length * bbdiag;
 
+    //////////////////////////////////
+    // store amips
+    auto amips_attribute =
+        mesh->register_attribute<double>("wildmeshing_amips", mesh->top_simplex_type(), 1);
+    auto amips_accessor = mesh->create_accessor(amips_attribute.as<double>());
+    // amips update
+    auto compute_amips = [](const Eigen::MatrixXd& P) -> Eigen::VectorXd {
+        assert(P.cols() == 3 || P.cols() == 4); // cols --> number of neighbors
```

Comment:
shouldn't it be "P.cols() == P.rows()+1" and assert (P.rows() == 2 || P.rows() == 3)